docs/expressions.md

```diff
@@ -330,6 +345,20 @@ Remove any values lower than or equal to number from a series. Will error if thi
 
 Remove any NaN or Inf values from a series. Will error if this operation results in an empty series.
 
+## epoch()
+
+Returns the Unix epoch in seconds of the expression start time (scalar).
+
+## filter(series, number)
+
+Returns all results in series that are a subset of anything in number, or
```

Comment:
I agree it is still a bit confusing. Can you add some examples? Is the second argument a series, a single double/int, or a slice of time/value pairs? If it is a series, how can I create them, since we don't have series literals.